docs/readthedocs/index.rst

```diff
@@ -10,13 +10,11 @@ Welcome to the documentation of simona - an agent-based discrete-event power sys
 Institute of Energy Systems, Energy Efficiency and Energy Economics at TU Dortmund University, Germany.
 
 .. toctree::
-   :maxdepth: 3
+   :maxdepth: 2
 
-   about
    usersguide
    config/config
```

Comment:
Would you might having the config section as part of the user's guide? I think, it is mandatory to run a simulation, right?

Reply:
Since config is empty I suggest to delete this part